client/src/test/java/com/kritsit/casetracker/shared/domain/model/IncidentTest.java

```diff
@@ -81,10 +81,8 @@ public void testEquals_Class() {
     }
 
     public void testEquals() {
-        LocalDate date = LocalDate.now();
```

Comment:
IncidentTest.java: "Dead store to local variable". The initial LocalDate.now() values were never used, did you want those there?